Source/DafnyCore/Pipeline/Compilation.cs

```diff
@@ -123,11 +133,19 @@ CompilationInput input
     }
 
     foreach (var uri in Options.CliRootSourceUris) {
-      var file = DafnyFile.CreateAndValidate(errorReporter, fileSystem, Options, uri, Token.Cli);
+      var shortPath = Path.GetRelativePath(Directory.GetCurrentDirectory(), uri.LocalPath);
+      var file = DafnyFile.CreateAndValidate(errorReporter, fileSystem, Options, uri, Token.Cli,
+        $"command-line argument '{shortPath}' is neither a recognized option nor a Dafny input file (.dfy,.doo,.toml).");
       if (file != null) {
         result.Add(file);
+      } else {
+        return result;
       }
     }
+    if (Options.UseStdin) {
+      var uri = new Uri("stdin:///");
+      result.Add(DafnyFile.CreateAndValidate(errorReporter, fileSystem, Options, uri, Token.Cli));
+    }
```

Comment:
You can't. `StdIn` is not an IDE option.

Reply:
Sure, I'm just pointing out that technically you could add `--stdin` to the Language Server Launch Args, and now it seems like it would actually do something rather than being ignored. I assume it's harmless but asking just in case.

Reply:
If you use the CLI to specify an option to a command to which it doesn't apply, then the CLI will return an error. If you pass it through the project file, the CLI won't error but it'll be ignored.